build_run_docker.sh

```diff
@@ -624,41 +587,40 @@ if [ $NUM_API_NODES -gt 0 ]; then
 
         template+="$VOLUMES_FIELD"
         if [ "$NUM_MYSQL_NODES" -gt 0 ]; then
-            add_volume_to_template "sysbench_single" "$BENCH_DIR/sysbench_single" $VOLUME_TYPE_BENCH_DIRS no
-            add_volume_to_template "dbt2_single" "$BENCH_DIR/dbt2_single" $VOLUME_TYPE_BENCH_DIRS no
+            add_volume_to_template "sysbench_single" "$BENCH_DIR/sysbench_single" no
+            add_volume_to_template "dbt2_single" "$BENCH_DIR/dbt2_single" no
             if [ "$NUM_MYSQL_NODES" -gt 1 ]; then
-                add_volume_to_template "sysbench_multi" "$BENCH_DIR/sysbench_multi" $VOLUME_TYPE_BENCH_DIRS no
-                add_volume_to_template "dbt2_multi" "$BENCH_DIR/dbt2_multi" $VOLUME_TYPE_BENCH_DIRS no
+                add_volume_to_template "sysbench_multi" "$BENCH_DIR/sysbench_multi" no
+                add_volume_to_template "dbt2_multi" "$BENCH_DIR/dbt2_multi" no
             fi
         fi
 
         # If we are using volumes for the benchmarking directories, we have to mount these files single-handedly.
         # They will then also be available inside the volumes.
-        if [ "$VOLUME_TYPE_BENCH_DIRS" == "docker" ]; then
-            template+="$BIND_DBT2_SINGLE_CONF_TEMPLATE"
-            template+="$BIND_DBT2_MULTI_CONF_TEMPLATE"
-            template+="$BIND_AUTOBENCH_SYS_SINGLE_TEMPLATE"
-            template+="$BIND_AUTOBENCH_SYS_MULTI_TEMPLATE"
-            template+="$BIND_AUTOBENCH_DBT2_SINGLE_TEMPLATE"
-            template+="$BIND_AUTOBENCH_DBT2_MULTI_TEMPLATE"
+        if [ "$VOLUME_TYPE" == "docker" ]; then
+            add_file_to_template "$DBT2_CONF_SINGLE_FILEPATH" "$BENCH_DIR/dbt2_single/dbt2_run_1.conf"
+            add_file_to_template "$DBT2_CONF_MULTI_FILEPATH" "$BENCH_DIR/dbt2_multi/dbt2_run_1.conf"
+            add_file_to_template "$AUTOBENCH_SYS_SINGLE_FILEPATH" "$BENCH_DIR/sysbench_single/autobench.conf"
+            add_file_to_template "$AUTOBENCH_SYS_MULTI_FILEPATH" "$BENCH_DIR/sysbench_multi/autobench.conf"
+            add_file_to_template "$AUTOBENCH_DBT2_SINGLE_FILEPATH" "$BENCH_DIR/dbt2_single/autobench.conf"
+            add_file_to_template "$AUTOBENCH_DBT2_MULTI_FILEPATH" "$BENCH_DIR/dbt2_multi/autobench.conf"
```

Comment:
This fails unless you use the `[ "$NUM_MYSQL_NODES" -gt 1 ]` check, such as this:

```
if [ "$VOLUME_TYPE" == "docker" ]; then
    add_file_to_template "$DBT2_CONF_SINGLE_FILEPATH" "$BENCH_DIR/dbt2_single/dbt2_run_1.conf"
    add_file_to_template "$AUTOBENCH_DBT2_SINGLE_FILEPATH" "$BENCH_DIR/dbt2_single/autobench.conf"
    add_file_to_template "$AUTOBENCH_SYS_SINGLE_FILEPATH" "$BENCH_DIR/sysbench_single/autobench.conf"
    if [ "$NUM_MYSQL_NODES" -gt 1 ]; then
        add_file_to_template "$DBT2_CONF_MULTI_FILEPATH" "$BENCH_DIR/dbt2_multi/dbt2_run_1.conf"
        add_file_to_template "$AUTOBENCH_DBT2_MULTI_FILEPATH" "$BENCH_DIR/dbt2_multi/autobench.conf"
        add_file_to_template "$AUTOBENCH_SYS_MULTI_FILEPATH" "$BENCH_DIR/sysbench_multi/autobench.conf"
    fi
fi
```

Reply:
Fixed, except I also added `if [ "$NUM_MYSQL_NODES" -gt 0 ]`